fdbserver/workloads/workloads.actor.h

```diff
@@ -103,25 +103,25 @@ struct KVWorkload : TestWorkload {
 	Key keyForIndex(uint64_t index, bool absent) const;
 };
 
-struct IWorkloadFactory {
-	static TestWorkload* create(std::string const& name, WorkloadContext const& wcx) {
+struct IWorkloadFactory : ReferenceCounted<IWorkloadFactory> {
```

Comment:
I just realized `IWorkloadFactory` should have a virtual destructor. We never delete these objects, so it might be fine as is, but it would be cleaner to add it.